DNN Platform/Tests/DotNetNuke.Tests.Data/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.SqlServer.Compact" version="4.0.8876.1" targetFramework="net45" />
```

Comment:
Needs ``allowedVersions="[4.0.8876.1]" `` to prevent problems in TeamCity builds,

Reply:
thanks, done